main.nf

```diff
@@ -892,6 +893,38 @@ process markDuplicates {
 }
 
 
+process qualimap {
+    label 'low_memory'
+    tag "${bam.baseName}"
+    publishDir "${params.outdir}/qualimap", mode: 'copy'
+
+    when:
+    !params.skip_qc && !params.skip_qualimap
+
+    input:
+    file bam from bam_qualimap
+    file gtf from gtf_qualimap.collect()
+
+    output:
+    file "${bam.baseName}" into qualimap_results
+
+    script:
+    def qualimap_direction = 'non-strand-specific'
```

Comment:
This is not used in the call?

Reply:
(e.g. you don't use it down in the script section)

Reply:
thanks! I'll fix it.